packages/cli/src/plugins/etherscan.ts

```diff
@@ -10,6 +10,7 @@ const apiUrls = {
   // Optimism
   [10]: 'https://api-optimistic.etherscan.io/api',
   [420]: 'https://api-goerli-optimistic.etherscan.io/api',
+  [11155420]: 'https://api-sepolia-optimistic.etherscan.io/api',
```

Comment:
```suggestion
  [11_155_420]: 'https://api-sepolia-optimistic.etherscan.io/api',
```

Reply:
Formatted the chainId to the above mentioned format.